src/languages/bash.js

```diff
@@ -55,11 +55,10 @@ export default function(hljs) {
       VAR
     ]
   };
-  const SHEBANG = {
-    className: 'meta',
-    begin: /^#![^\n]+sh\s*$/,
+  const SHEBANG = hljs.SHEBANG({
+    binary: /(fish|bash|zsh|sh)/,
```

Comment:
Then the relevance would have to change.. not every binary ending in `sh` is a shell...

Reply:
I'm not sure what are the most frequently used shells (except for bash and historically sh, of course). Probably your list is good enough, I just don't know.

Reply:
If you'd like to suggest a few more to add, we have the room. :-)  I'd rather add them slowly than have false positives.  And if your concern is just highlighting the shebang itself (vs treating it as a comment) we could add a basic SHEBANG rule also with 0 relevancy...

Reply:
I didn't look at that as a "match any *sh" I read that as match the path then `sh`... but you're right that before it would have been more generic.  I guess I just dislike that - at least when it comes to appointing 10 relevancy.

Reply:
> If you'd like to suggest a few more to add, we have the room. :-)

I've heard of, say, `ksh`, but I have no idea if someone use it these days. I have no must-have suggestions.

> And if your concern is just highlighting the shebang itself (vs treating it as a comment) we could add a basic SHEBANG rule also with 0 relevancy - to catch shell like files with an unknown binary...

I think it's a good idea.

Reply:
I expanded the list: ```    "fish",
    "bash",
    "zsh",
    "sh",
    "csh",
    "ksh",
    "tcsh",
    "dash",
    "scsh",
```